Makefile

```diff
@@ -255,6 +255,9 @@ validate_namespace:
 
 run: validate_namespace deploy_assisted_service deploy_ui
 
+run_onprem:
+	DEPLOY_TARGET=onprem make validate_namespace deploy_assisted_service deploy_ui
```

Comment:
``validate_namespace deploy_assisted_service deploy_ui`` is the same as ``run``, no?
In general, I think we'd better have less number of makefile targets, as it's gets harder to maintain a larger number of them
so ``make run DEPLOY_TARGET=onprem`` should be simple enough imho

Reply:
I remember changing it to `DEPLOY_TARGET=onprem make run`
But I wondered about this too.